src/wincng.c

```diff
@@ -253,6 +253,17 @@ _libssh2_wincng_init(void)
         }
     }
 
+    ret = BCryptOpenAlgorithmProvider(&_libssh2_wincng.hAlgAES_ECB,
+        BCRYPT_AES_ALGORITHM, NULL, 0);
+    if (BCRYPT_SUCCESS(ret)) {
+        ret = BCryptSetProperty(_libssh2_wincng.hAlgAES_ECB, BCRYPT_CHAINING_MODE,
+            (PBYTE)BCRYPT_CHAIN_MODE_ECB,
+            sizeof(BCRYPT_CHAIN_MODE_ECB), 0);
+        if (!BCRYPT_SUCCESS(ret)) {
```

Comment:
Please align the lines with the opening ( , just like in lines 249 and 250. Otherwise this patch looks really good. I am looking forward to testing and trying it soon.